app/app/explore/components/ExploreTable.tsx

```diff
@@ -0,0 +1,359 @@
+                />
+                <span className="ml-3 font-medium text-sm md:text-base">
+                  {getAssetSymbol(pool.asset)}
+                </span>
+              </div>
+            ),
+          },
+          {
+            key: "volume",
+            label: "Volume (24h)",
+            width: "w-1/4",
+            render: (pool) =>
+              addDollarSignAndSuffix(calculateVolumeUSD(pool, runePriceUSD!)),
+          },
+          {
+            key: "volumeDepth",
+            label: "Volume/Depth",
+            width: "w-1/4",
+            render: (pool) =>
+              formatNumber(
+                calculateVolumeDepthRatio(pool, runePriceUSD!),
+                2,
+                2,
+              ),
+          },
+          {
+            key: "tvl",
+            label: "TVL",
+            width: "w-1/4",
+            sortable: true,
+            render: (pool) => getFormattedPoolTVL(pool, runePriceUSD!),
+          },
+          {
+            key: "apr",
+            label: "APR",
+            width: "w-1/4",
+            sortable: true,
+            render: (pool) =>
+              `${formatNumber(parseFloat(pool.poolAPY) * 100, 2, 2)}%`,
+          },
+        ]
+      : [
+          {
+            key: "asset",
+            label: "Asset",
+            width: "w-48 md:w-1/3",
+            render: (saver) => (
+              <div className="flex md:items-center">
+                <Image
+                  src={getLogoPath(saver.asset)}
+                  alt={`${getAssetSymbol(saver.asset)} logo`}
+                  width={28}
+                  height={28}
+                  className="rounded-full"
+                />
+                <span className="ml-3 font-medium text-sm md:text-base">
+                  {getAssetSymbol(saver.asset)}
+                </span>
+              </div>
+            ),
+          },
+          {
+            key: "saversCount",
+            label: "Savers",
+            width: "w-1/4",
+            render: (saver) => saver.saversCount,
+          },
+          {
+            key: "filled",
+            label: "Utilization",
+            width: "w-1/4",
+            render: (saver) => `${formatNumber(saver.filled * 100, 2, 2)}%`,
+          },
+          {
+            key: "tvl",
+            label: "TVL",
+            width: "w-1/4",
+            sortable: true,
+            render: (saver) => getFormattedSaverTVL(saver),
+          },
+          {
+            key: "apr",
+            label: "APR",
+            width: "w-1/4",
+            sortable: true,
+            render: (saver) =>
+              `${formatNumber(parseFloat(saver.saversReturn) * 100, 2, 2)}%`,
+          },
+        ];
+
+  const sortedData = useMemo(() => {
+    const items = [...data];
+    items.sort((a, b) => {
+      if (sortConfig.key === SortKey.TVL) {
+        const aTvl =
+          type === "pools"
+            ? calculatePoolTVL(a as unknown as PoolDetail, runePriceUSD!)
```

Comment:
Having separate Savers and Pool components with concrete types would avoid having to do type coercion here. This is also bad practice because it obfuscates errors.